arch/arm/core/cortex_a_r/smp.c

```diff
@@ -240,8 +240,10 @@ void sched_ipi_handler(const void *unused)
 }
 
 /* arch implementation of sched_ipi */
-void arch_sched_ipi(void)
+void arch_sched_ipi(uint32_t cpu_bitmap)
 {
+	ARG_UNUSED(cpu_bitmap);
+
 	broadcast_ipi(SGI_SCHED_IPI);
```

Comment:
The implementation of this function has a loop over CPUs doing gic_raise_sgi() already, so surely this arch can do this.  If you don't want to muck with ARM code directly, maybe @carlocaione @npitre et. al. might have input?   But for sure we want to get ducks in a row before merge, either with code in the PR or an active issue to add it.

Reply:
Can do.